src/gui/raster/qgsrasterlayerproperties.h

```diff
@@ -305,5 +313,11 @@ class GUI_EXPORT QgsRasterLayerProperties : public QgsOptionsDialogBase, private
     QgsCoordinateReferenceSystem mBackupCrs;
 
     QgsRasterAttributeTableWidget *mRasterAttributeTableWidget = nullptr;
+
+    void initMapTipPreview();
+
+    QWidget *mMapTipPreviewWidget = nullptr;
+    QgsWebView *mMapTipPreview = nullptr;
+    const int MARGIN_VALUE = 5;
```

Comment:
Maybe static const and initialize in cpp file.

Reply:
I can make it static, but I believe most` static const int` in the code base are initialized in the header. Anyway, if this PR is accepted, I intend to propose a new PR that makes the margin, border and background color configurable, and make it possible to have semi-transparent map tips, so MARGIN_VALUE would disappear.